RedemptionQuick/sets/carddata.txt

```diff
@@ -1823,7 +1823,7 @@ Philistine Chariot and Horses	Thesaurus ex Preteritus	Philistine_Chariot_and_Hor
 Philistine Garrison	Thesaurus ex Preteritus	Philistine_Garrison_(TxP)	Evil Character	Black	12	8	Warrior	Generic, Philistine, Fought Earthly battle	Immune to lone Heroes.	Fixed	I Samuel 14:12	
 Philistine Outpost	Thesaurus ex Preteritus	Philistine_Outpost_(TxP)	Fortress	Evil					When you are attacked, you may discard this card or an evil card from deck to search discard pile for a generic Philistine and place it in your territory. Protect Sites from being placed beneath decks.	Fixed	I Samuel 14:12	
 Philistine Priests	Priests	Philistine_Priests_(Pi)	Evil Character	Black	5	5		Generic Priest	You may search draw pile for Temple of Dagon and put it in play.  If another Philistine Evil Character is in play, you may negate and deactivate one O.T.  Artifact in play.	Common	I Samuel 6:2	
-Philosophy	Promo	Philosophy_(good)_(Promo)	DAE	Multi	2-Mar	3-Feb				Promo	I Corinthians 1:20,25 (Good); Proverbs 1:29, 32 (Evil)	
+Philosophy	Promo	Philosophy_(good)_(Promo)	DAE	Multi	3	2				Promo	I Corinthians 1:20,25 (Good); Proverbs 1:29, 32 (Evil)	
```

Comment:
Philosophy should have Strength  3/2  and Toughness 2/3

Reply:
Corrected